pims/tests/test_common.py

```diff
@@ -38,16 +38,14 @@ def _skip_if_no_ImageIO():
 
 
 def _skip_if_no_libtiff():
-    try:
-        import libtiff
-    except ImportError:
+    import pims.tiff_stack
```

Comment:
Right, I am not saying that we are consistent ;-) It's your call, I am fine with merging this.